src/data/linux-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const linuxQuiz = [
+  {
+    Question: "In Linux, which command option is used to perform a case-insensitive search when searching for text patterns?",
+    Answer: "grep -i",
+    Distractor1: "grep -c",
+    Distractor2: "grep -v",
+    Distractor3: "grep -s",
+    Explanation:
+      "grep -i command is used to perform a case-insensitive search, allowing you to find text patterns ignoring the case.",
```

Comment:
small grammar change 

```suggestion
      "The grep -i command is used to perform a case-insensitive search, allowing you to find text patterns ignoring the case.",
```